setup.py

```diff
@@ -46,43 +46,42 @@
         'pytest-runner~=4.0',
     ],
     install_requires=[
-        'Jinja2~=3.0.3',
-        'Flask~=2.0.3',
-        'Flask-Compress~=1.10.1',
-        'Flask-Cors~=3.0.10',
-        'Flask-Login~=0.5.0',
-        'Werkzeug~=2.0.0',
-        'pyasn1~=0.4.6',
-        'google-api-python-client~=2.33.0',
-        'google-auth~=2.29.0',
-        'google-auth-httplib2~=0.1.1',
+        'Jinja2~=3.1.2',
+        'Flask~=3.0.0',
+        'Flask-Compress~=1.15',
+        'Flask-Cors~=5.0.1',
+        'Flask-Login~=0.6.3',
+        'Werkzeug~=3.0.0',
+        'pyasn1~=0.6.1',
+        'google-api-python-client~=2.167.0',
+        'google-auth~=2.34.0',
+        'google-auth-httplib2~=0.2.0',
         'google-compute-engine~=2.8.13',
         'rauth~=0.7',
-        'pysam~=0.21.0',
-        'marisa-trie~=0.7',
-        'intervaltree~=2.1',
-        'tqdm~=4.14',
-        'openpyxl~=2.5',
-        'scipy~=1.0',
-        'numpy~=1.14',
-        'requests[security]~=2.18',
-        'cryptography~=3.2',
-        'idna~=2.6',
-        'gunicorn~=19.7',
-        'boltons~=18.0',
-        'blist~=1.3.6',
-        'cffi~=1.11',
+        'pysam~=0.23.0',
+        'marisa-trie~=1.2.1',
+        'intervaltree~=3.1.0',
+        'tqdm~=4.67.0',
+        'openpyxl~=3.1.5',
+        'scipy~=1.15.2',
+        'numpy~=2.2.5',
+        'requests[security]~=2.32.3',
+        'cryptography~=44.0.2',
+        'idna~=3.10',
+        'gunicorn~=23.0.0',
+        'boltons~=25.0.0',
+        'cffi~=1.17.1',
         'wget~=3.2',
-        'elasticsearch~=6.2.0',
+        'elasticsearch~=9.0.0',
         'latex~=0.7.0',
         'attrs',
-        'pandas~=0.24.2',
+        'pandas~=2.2.3',
         'SQLAlchemy~=1.3.19',
```

Comment:
@hafa786 This should be updated to sqlalchemy 1.4.54, which is the newest bugfix in 1.x version https://pypi.org/project/SQLAlchemy/1.4.54/ , it should be compatible with our current code